PaymentHighway/UI/Theme/DefaultTheme.swift

```diff
@@ -5,23 +5,24 @@
 //  Copyright © 2018 Payment Highway Oy. All rights reserved.
 //
 
-private let defaulBackgroundColor = UIColor.white
-private let defaultPrimaryForegroundColor = UIColor(hexInt: 0x808080)
-private let defaultPrimaryActiveForegroundColor = UIColor(hexInt:  0x000000)
-private let defaultSecondaryForegroundColor = defaultPrimaryForegroundColor
-private let defaultSecondaryActiveForegroundColor = defaultPrimaryActiveForegroundColor
-private let defaultErrorForegroundColor = UIColor(hexInt: 0xe80f0f)
-private let defaultErrorActiveForegroundColor = UIColor(hexInt: 0x993333)
-private let defaultHighlightColor = UIColor(hexInt: 0x007AFF)
-private let defaultHighlightDisableColor = UIColor.gray
+private let defaultBarTintColor =  UIColor(hexInt: 0xf6f6f6)
+private let defaultBackgroundColor = UIColor.white
+private let defaultPrimaryForegroundColor = UIColor(red: 0, green: 0, blue: 0, alpha: 153)
+private let defaultPrimaryActiveForegroundColor = UIColor(red: 0, green: 0, blue: 0, alpha: 222)
+private let defaultSecondaryForegroundColor = UIColor(hexInt: 0x9d00ba)
+private let defaultSecondaryActiveForegroundColor = defaultSecondaryForegroundColor
+private let defaultErrorForegroundColor = UIColor.red
+private let defaultErrorActiveForegroundColor = UIColor.red
+private let defaultHighlightColor = defaultSecondaryForegroundColor
+private let defaultHighlightDisableColor = UIColor(red: 124, green: 124, blue: 124, alpha: 153)
 
-private let defaulRoundedBorderRadius: CGFloat = 20
-private let defaultPlaceholderFontScale: CGFloat = 0.7
-private let defaulBorderWidth: CGFloat = 1.5
-private let defaulPlaceholderAnimationDuration: Double = 0.25
-private let defaultFontSize: CGFloat = 13.0
-private let defaultTextImages: [TextFieldType] = [.cardNumber, .expiryDate, .securityCode]
-private let defaultTextPaddingX: CGFloat = 20.0
+private let defaultRoundedBorderRadius: CGFloat = 28
+private let defaultPlaceholderFontScale: CGFloat = 0.8
+private let defaultBorderWidth: CGFloat = 1.5
+private let defaultPlaceholderAnimationDuration: Double = 0.25
```

Comment:
Could infer `Double` automatically, so just `= 0.25` would suffice?

Reply:
yes this in inferred automatically. removed. thanks